src/main/cpp/subsystems/Chassis.cpp

```diff
@@ -0,0 +1,42 @@
+#include "subsystems/Chassis.h"
+#include "RobotMap.h"
+#include "commands/TankDrive.h"
+
+const char Chassis::kSubsystemName[] = "Chassis";
+
+std::shared_ptr<Chassis> Chassis::self;
+
+std::shared_ptr<Chassis> Chassis::getInstance() {
+  if (! self) {
+     self = std::shared_ptr<Chassis>(new Chassis);
+  }
+  return self;
+}
+
+
+
+Chassis::Chassis() : Subsystem(kSubsystemName),
+  //defaultCommand(nullptr),
+  right1Wheel{RobotMap::kIDRight1Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  right2Wheel{RobotMap::kIDRight2Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  left1Wheel{RobotMap::kIDLeft1Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  left2Wheel{RobotMap::kIDLeft2Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless}
+  {
+    left2Wheel.Follow(left1Wheel);
+	  right2Wheel.Follow(right1Wheel);
+    right1Wheel.SetInverted(true);
+    right2Wheel.SetInverted(true);
+  }
+
+void Chassis::InitDefaultCommand() {
+  // Set the default command for a subsystem here.
+  SetDefaultCommand(new TankDrive);
+}
+
+void Chassis::SetTankDrive(double left, double right) {
+  left1Wheel.Set(left);
+  right1Wheel.Set(right);
+  left2Wheel.Set(left);
+  right2Wheel.Set(right);
```

Comment:
Don't call `Set` for `left2Wheel` and `right2Wheel`. These controllers are already followers.